### ISSUE TITLE
Prediction results lack discrimination when running the tutorial example

### ISSUE THREAD

**Issue by wangjiawen2013** [chemprop/chemprop#1393]
Hi,

I'm following the official tutorial (https://chemprop.readthedocs.io/en/latest/predicting_regression_reaction.html). However, the predictions on the test set are all squeezed into a narrow range near the mean, with almost no differentiation between samples and the data range differs a lot between targets and predictions.

Is this expected behavior for this tutorial, or is it just a demo missing some critical preprocessing step? I also observed similar results for some other chemprop tutorial

<img width="1071" height="651" alt="Image" src="https://github.com/user-attachments/assets/45fdf4b4-2c8a-4deb-8b23-57efdadd1718" />

The correlation coefficient is not high:

<img width="698" height="350" alt="Image" src="https://github.com/user-attachments/assets/1be3382d-fdb0-4c06-bc48-637c0439fcfd" />

**Comment by JacksonBurns** [comment]
Hi @wangjiawen2013, you are smart to check these things.

The models in the tutorials are trained for very few epochs and on very small amounts of data, mostly so that they run quickly on our testing suite. If you increase either it should reduce the issue.

**Comment by KnathanM** [comment]
For example, see this line of code that was used to make the model file used in the reaction prediction tutorial.
https://github.com/chemprop/chemprop/blob/9a0b47ad19ebf440c1557787e666d442728adaad/tests/regenerate_models.sh#L69

I'll close the issue for now, but if you have further questions, feel free to continue the conversation.

**Comment by wangjiawen2013** [comment]
CheMeleon Foundation Finetuning improved significantly when using the same epoch=20.
